python/ray/tune/suggest/bohb.py

```diff
@@ -0,0 +1,77 @@
+"""BOHB (Bayesian Optimization with HyperBand)"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from hpbandster import BOHB
```

Comment:
can you do a soft import here? like in `suggest/ax.py`; and correspondingly raise an error in __init__ if not found?